packages/api/src/utils/client/httpClient.ts

```diff
@@ -394,6 +396,10 @@ export class HttpClient implements IHttpClient {
       abortSignals.forEach((s) => s?.removeEventListener("abort", onSignalAbort));
     }
   }
+
+  private getRequestMethod(requestFormat: `${WireFormat}`): typeof this._request {
```

Comment:
that is this `${}` syntax in the method signature? First I've seen it used there

Reply:
If we use the enum type it is too strict as it only allows to pass values from the enum and not the same value as string literal

same type as we have for request options
https://github.com/ChainSafe/lodestar/blob/6cd7f08a4d0042f53e5a8808b2bc126a06980927/packages/api/src/utils/client/request.ts#L15

This allows the following

```ts
await api.beacon.getGenesis({responseWireFormat: "ssz"})

// instead of having to do
await api.beacon.getGenesis({responseWireFormat: WireFormat.ssz})
```

and you also get nice intellisense
![image](https://github.com/ChainSafe/lodestar/assets/38436224/c02bef81-a1e9-42fb-83a5-5a8c81474836)


More or less just a devex improvement